frontend/src/static/js/v_authorship.js

```diff
@@ -40,21 +41,40 @@ window.vAuthorship = {
       totalBlankLineCount: '',
       filesSortType: 'lineOfCode',
       toReverseSortFiles: true,
-      activeFilesCount: 0,
+      isActive: true,
       filterSearch: '*',
       sortingFunction: (a, b) => -1 * window.comparator(filesSortDict.lineOfCode)(a, b),
-      isSearchBar: false,
-      isCheckBoxes: true,
     };
   },
 
   watch: {
+    selectedFileTypes() {
+      if (this.selectedFileTypes.length === this.fileTypes.length) {
+        this.isSelectAllChecked = true;
+      }
+      if (this.selectedFileTypes.length === 0) {
+        this.isSelectAllChecked = false;
+      }
+    },
     filesSortType() {
       this.sortFiles();
     },
     toReverseSortFiles() {
       this.sortFiles();
     },
+    selectedFiles() {
+      setTimeout(this.updateCount, 0);
```

Comment:
Last question, what does this do?

Reply:
To drag the `updateCount` function to the end of the event queue so we render the file messages first. I kept it from the previous code, not very sure if it is still relevant now.

Reply:
OK